quarkus/tests/integration/src/test/java/org/keycloak/it/cli/dist/GelfRemovedTest.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2023 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.keycloak.it.cli.dist;
+
+import io.quarkus.test.junit.main.Launch;
+import io.quarkus.test.junit.main.LaunchResult;
+import org.junit.jupiter.api.Test;
+import org.keycloak.it.junit5.extension.CLIResult;
+import org.keycloak.it.junit5.extension.DistributionTest;
+import org.keycloak.it.junit5.extension.RawDistOnly;
+import org.keycloak.quarkus.runtime.cli.command.StartDev;
+
+import static org.keycloak.config.LoggingOptions.GELF_ENABLED;
+
+@DistributionTest
+@RawDistOnly(reason = "Verifying the help message output doesn't need long spin-up of docker dist tests.")
+public class GelfRemovedTest {
+
+    @Test
+    @Launch({ StartDev.NAME, "--help-all" })
+    void checkGelfRemoved(LaunchResult result) {
+        CLIResult cliResult = (CLIResult) result;
+        if ("true".equals(System.getProperty("includeGelf"))) {
+            cliResult.assertMessage("gelf");
```

Comment:
I like the simpler approach. Just for the clarification (I might've missed something) - does this always test only the "true" branch of the if clause?

Don't you wan't then rather to add a condition based on whether the `product` property is used?

Other than that, if docs get an improvement, then LGTM. Good work,  @mabartos 😊.

Reply:
> does this always test only the "true" branch of the if clause?

For community, yes. For product, the `includeGelf` is not specified[1], so the condition is evaluated to false. 

> Don't you wan't then rather to add a separate test case based when the product property is used?

AFAIK, it's not so simple. As that is our "dist" test, we're currently working with the already-built distribution located in the `quarkus/dist` module. See here[2].

[1] https://github.com/keycloak/keycloak/pull/22615/files#diff-d27c2ff3bd1df4ecd00219ceed139feb064a31f5f14a398c6f7a154262a1e57eR169-R171 
[2] https://github.com/keycloak/keycloak/blob/main/quarkus/tests/junit5/src/main/java/org/keycloak/it/utils/RawKeycloakDistribution.java#L372

Reply:
Got it, thanks for the clarification.